app/src/androidTest/java/fiofoundation/io/fiokotlinsdktestapp/TestNetSdkTests.kt

```diff
@@ -29,17 +30,17 @@ class TestNetSdkTests {
 
     private val baseUrl = "https://testnet.fioprotocol.io:443/v1/"
 
-    private var alicePrivateKey = "your private key"
-    private var alicePublicKey = "your public key"
-    private var bobPrivateKey = "your private key"
-    private var bobPublicKey = "your public key"
+    private var alicePrivateKey = ""
+    private var alicePublicKey = ""
+    private var bobPrivateKey = ""
+    private var bobPublicKey = ""
 
     private val testPrivateKey = "5Kbb37EAqQgZ9vWUHoPiC2uXYhyGSFNbL6oiDp24Ea1ADxV1qnu"
     private val testPublicKey = "FIO5kJKNHwctcfUM5XZyiWSqSTM5HTzznJP9F3ZdbhaQAHEVq575o"
     private val testMnemonic = "valley alien library bread worry brother bundle hammer loyal barely dune brave"
 
-    private var aliceFioAddress = "your_registered_address1@fiotestnet"
```

Comment:
Setting the address variables was put there to remind the developer what needs to go there to run the tests. Why remove it?

Reply:
I put these back into place